packages/internals/src/cli/checkUnsupportedDataProxy.ts

```diff
@@ -62,7 +60,7 @@ async function checkUnsupportedDataProxyMessage(command: string, args: Args, imp
 
       const datamodel = await fs.promises.readFile(argValue, 'utf-8')
       const config = await getConfig({ datamodel, ignoreEnvVarErrors: true })
-      const url = command === 'studio' ? config.datasources[0]?.url : getEffectiveUrl(config.datasources[0])
+      const url = getEffectiveUrl(config.datasources[0])
```

Comment:
Before this explicitly skipped `studio` for the usage of the `directUrl` (via `getEffectiveUrl`).
Now `studio` does not get special handling, which allows the command to execute with a `prisma://` `url` and a direct `directUrl` value.